### DISCUSSION TITLE
How many miles did a frequent flyer recently earn on a record-breaking flight?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#571] [Galaxy Trivia] [answered]
**Bonus:** How long did it take him to earn it?

**Comment by peterbaccaro** [comment, marked as answer]
Tom Stuker had, as of July 2022, clocked up **23 million miles** on over 12,000 flights.

By 2009 he had 10 million miles which had taken 29 years to accumulate.
Therefore by 2022 when he clocked up **23 million miles** that would have taken him **42** years.
